theotime2005/bnote-settings
# [MINORFEATURE] Add the time zone parameter

## Problem

In the B.note, a new parameter has been added [See the pr here](https://github.com/devel-erb/bnote/pull/21). This is a parameter to define the time zone of the B.note.

## Solution

Add this parameter with the list of different time zones via [the API](https://timeapi.io) used in the B.note.

## Note

A specific parameter `isTranslate` has been added to indicate whether labels should be translated or not.

## Test

- Go to the management page,

- Display the default file,

- Check the presence of the time zone in the `User Interface` section,

- Make sure there is no problem with the translation.
